README.md

```diff
@@ -173,6 +173,31 @@ Font files should be included in the cookbooks
 windows_font 'Code New Roman.otf'
 ```
 
+### windows_http_acl
+Sets the Access Control List for an http URL to grant non-admin accounts permission to open HTTP endpoints
+
+#### Actions
+- :create: creates or updates the ACL for a URL
```

Comment:
Can you leave a comment mentioning that acls granted to other users will be removed?